assets/css/fonts.css

```diff
@@ -40,6 +40,20 @@
     src: url('/fonts/ExpensifyMono-Bold.woff2') format('woff2'),  url('/fonts/ExpensifyMono-Bold.woff') format('woff');
 }
 
+@font-face {
+    font-family: ExpensifyNewKansas-Medium;
```

Comment:
The font-family here should just be `ExpensifyNewKansas`. Medium, or Medium Italic, are just weights/styles of the font family.

Reply:
We can get close to this! iOS uses `Expensify New Kansas` (with spaces) and I'm not sure if we can change this since it's the name stored in the font file.

For android, we'll have to switch to [XML fonts](https://github.com/jsamr/react-native-font-demo#android) and specify each combo of font-weight/font-style. Otherwise, android uses the file names and using a different value as font-family would break the font.

I pushed the changes to make this happen. Let me know if you still have any concerns!

Reply:
@shawnborton @roryabraham and I discussed this and we agreed that it's best to tackle all the font-family grouping in a follow-up PR for a couple of reasons:

1. If we do it here, it would introduce yet another way of importing fonts, which touches native android code - we don't want that
2. We can implement a better solution straight in the library we use to link assets, making it easier to import fonts in the future

Reply:
I'll keep these font-families as is in favor of [this issue](https://github.com/Expensify/App/issues/13772)

Reply:
I don't think I am following, it's really just a matter of changing the names around no?

Reply:
Unfortunately not. Doing so breaks the font on android as android uses the font filename, i.e. ExpensifyNewKansas-Medium, etc as `fontFamily` and renaming the files would break iOS since iOS needs the filename to match the font's postscript name.

As I explained above, we can get around this by using [XML fonts](https://github.com/jsamr/react-native-font-demo#android) for android and that's what the issue I linked above will do!

Reply:
Got it got it, thanks for clarifying! Okay, then yeah let's get this implemented and you can follow up and fix all of the fonts :)

Reply:
This was a big gotcha for me on the fonts PR too.